daktari/check.py

```diff
@@ -96,7 +96,7 @@ def validate_semver_expression(
                 f"{application} version is {installed_version}, {recommended_version} recommended"
             )
 
-        return self.passed(f"{application} version is {required_version}")
+        return self.passed(f"{application} version is {installed_version}")
```

Comment:
(I changed this because it's unhelpful when no `required_version` is specified and the detected version might be more useful than the required/recommended versions in this context anyway).